terraform/deploy/internet_egress.tf

```diff
@@ -0,0 +1,54 @@
+resource "aws_ec2_transit_gateway_vpc_attachment" "concourse" {
+  subnet_ids         = module.vpc.outputs.aws_subnets_private.*.id
+  transit_gateway_id = data.terraform_remote_state.internet_egress.outputs.internet_transit_gateway.id
+  vpc_id             = module.vpc.outputs.aws_vpc.id
+  tags = merge(
+    local.tags,
+    map(
```

Comment:
you can make this cleaner, see example on:
https://git.ucd.gpn.gov.uk/dip/aws-common-infrastructure/wiki/Use-of-Tags#terraform-012-ec2-tag-example

Reply:
Thanks for the pointer. Fixed.